src/Context/BasicContext.php

```diff
@@ -55,6 +60,50 @@ class BasicContext implements Context
      */
     protected $datetimeFormat = 'Y-m-d H:i:s';
 
+    /**
+     * @var FixtureContext
+     */
+    protected $fixtureContext = null;
+
+    /**
+     * Get the fixture context of the current module
+     *
+     * @BeforeScenario
+     */
+    public function gatherContexts(BeforeScenarioScope $scope): void
+    {
+        /** @var InitializedContextEnvironment $environment */
+        $environment = $scope->getEnvironment();
+
+        // Find the FixtureContext defined in behat.yml
+        $subClasses = $this->getSubclassesOf(FixtureContext::class);
+        foreach ($subClasses as $class) {
+            if (!$environment->hasContextClass($class)) {
+                continue;
+            }
+            $this->fixtureContext = $environment->getContext($class);
+            break;
+        }
+        // Fallback to base FixtureClass
+        if (!$this->fixtureContext && $environment->hasContextClass(FixtureContext::class)) {
+            $this->fixtureContext = $environment->getContext(FixtureContext::class);
+        }
+    }
+
+    /**
+     * Gets the subclasses of a class
+     */
+    private function getSubclassesOf($parent): array
```

Comment:
There already exists `ClassInfo::subclassesFor()` so this would duplicate the mechanism.

Reply:
I'm not sure ... looks like [`ClassInfo::subclassesFor()`](https://github.com/silverstripe/silverstripe-framework/blob/4b8bc55c4052e2fe4b22ccfe14d76adfda41cf81/src/Core/ClassInfo.php#L189) does something substantially different. Then again it might land you in the same spot.

It's worth nothing that this method is private, so we can nuke it later on if we decide we don't need it.